### PR TITLE
Labware/add labcyte 384 pp std v

### PR DESCRIPTION
**Pull request by m-i-sadowski**
This PR adds one supported plate type for the Labcyte echo - the standard volume 384 plate used by their system

FYI @ddn0

### CODE REVIEW

**Comment by m-i-sadowski** [comment]
If one of you (Xander most likely) can check the description I've forwarded on vs. the addition in the library it'd be a good idea.

File: inventory/testinventory/make_plate_library.go [hunk 1]
```diff
@@ -830,3 +831,52 @@ func make96DeepWellLowVolumePlate() *wtype.LHPlate {
...
+	zstartOffsetCorrection := 2.5
+
+	plateName := "Labcyte_384PP_StdV"
+	wellName := "Labcyte_384PP_StdV"
+	manufacturer := "Labcyte"
+
+	numberOfRows := 16
+	numberOfColumns := 24
+
+	wellShape := "squre cylinder"
```
Comment (x-anderson) [inline inventory/testinventory/make_plate_library.go]: "square cylinder" - spellcheck to allow future proofing/sorting

File: inventory/testinventory/make_plate_library.go [hunk 2]
```diff
@@ -830,3 +831,52 @@ func make96DeepWellLowVolumePlate() *wtype.LHPlate {
 
 	return plate
 }
+
```
Comment (x-anderson) [inline inventory/testinventory/make_plate_library.go]: Is it possible to include the URL to the place where you got the dimensions from, so easy to reference if needed?

File: inventory/testinventory/make_plate_library.go (hunk 1 above)
Comment (m-i-sadowski) [reply inventory/testinventory/make_plate_library.go]: Fixed this - good spot  thanks!

File: inventory/testinventory/make_plate_library.go (hunk 2 above)
Comment (m-i-sadowski) [reply inventory/testinventory/make_plate_library.go]: I've added the link to the PDF from labcyte in the comment above the method.

**Pull request merged**